platform-server-kernel/src/main/resources/db/migration/20160325_120600__taskresult.sql

```diff
@@ -0,0 +1,29 @@
+DROP TABLE IF EXISTS `result_starttimes`;
```

Comment:
not yet mergeable because this does not copy existing data. @Exordian help needed. Should he just create the table? We will drop production db anyway